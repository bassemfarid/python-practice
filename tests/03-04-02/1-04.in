0
0

-10
10

-20
-30

5
-5

99
1
