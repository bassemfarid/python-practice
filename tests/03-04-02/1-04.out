The sum is: 0
The sum is: 0
The sum is: -50
The sum is: 0
The sum is: 100
I'm tired, goodbye
